decisions/2024-02-28_file_sync.md

```diff
@@ -0,0 +1,91 @@
+# File Sync
+
+- _Date_: 2024-02-28
+- _Deciders_: James Brunskill, Mark Prins, Andrei Evguenov, Chris Petty
+- _Status_: DECIDED (almost)
+- _Outcome_: Option 1
+
+## Context
+
+Our cold chain equipment management features require a mechanism to upload and download documents, and images related to assets.
+
+If these were to go via our standard sync process, large files would potentially block the sync process, slow down deployments to sites, etc.
+
+All options assume we'll use the open-mSupply Central Server Sync process recently established.
```

Comment:
Is this documented anywhere?  It sounds like this is designed not to sync to Legacy mSupply sites (but how is that possible)?

This feature has been a long-standing request - see  [Attach document to internal order and sync it to the supplying store requisition #11288](https://github.com/msupply-foundation/msupply/issues/11288)

Reply:
Thanks Mark,

@andreievg do you have any documentation for the open-mSupply central server sync design.

The quick version is that we plan to run a open-mSupply central server next to our existing msupply central server. open-mSupply will sync to legacy, but also sync to the open-mSupply central for new things such as files to avoid work being done in two places. However this does unfortunately mean this solution would only work for open-mSupply not legacy.

Reply:
> The quick version is that we plan to run a open-mSupply central server next to our existing msupply central server. open-mSupply will sync to legacy, but also sync to the open-mSupply central for new things such as files to avoid work being done in two places.

OK, so this is what is meant by the 'open-mSupply Central Server Sync' process.  Now I understand, it it makes good sense.